mlir-requirements.txt

```diff
@@ -0,0 +1 @@
+1ad9d6df32acc6d29d58e8ed6710e36746d0a4d6
```

Comment:
This should be `ROCmSoftwarePlatform/rocMLIR@1ad9d6df32acc6d29d58e8ed6710e36746d0a4d6 -DBUILD_FAT_LIBROCKCOMPILER=On`.

Reply:
Done